lib/_debugger.js

```diff
@@ -949,6 +950,12 @@ Interface.prototype.controlEval = function(code, context, filename, callback) {
       }
     }
 
+    // exec process.title => exec("process.title");
+    var match = code.match(/exec\s+([^\n]*)/);
```

Comment:
This should probably use an anchoring match (i.e. `/^\s*exec`) so it won't get confused by e.g. `"exec exec exec"` as the input.